dipy/reconst/ivim.py

```diff
@@ -713,7 +727,7 @@ def cvx_fit(self, signal, phi):
         constraints = [cvxpy.sum(f) == 1,
                        f[0] >= 0.011,
                        f[1] >= 0.011,
-                       f[0] <= 0.29,
+                       f[0] <= self.bounds[1][0],
```

Comment:
What are the other hard-coded values here?

Reply:
This was setup according to the bounds used in the MIX paper, for other models.

Reply:
OK. I guess we can leave the hard-coded values for now. 

Could you please add a comment that points out where they are from?